### PR TITLE
Added metrics

### PR DESCRIPTION
**Pull request by superbeagle**
Added root-process-instance-start, task-worker, and executed-decision-instances to metrics REST API metric names.

### CODE REVIEW

File: content/reference/rest/metrics/get-metrics-interval.md [hunk 1]
```diff
@@ -40,11 +40,14 @@ GET `/metrics`
...
       <code>executed-decision-elements</code>,
+      <code>executed-decision-instances</code>,
       <code>history-cleanup-removed-process-instances</code>,
       <code>history-cleanup-removed-case-instances</code>,
       <code>history-cleanup-removed-decision-instances</code>,
       <code>history-cleanup-removed-batch-operations</code>,
-      <code>history-cleanup-removed-task-metrics</code>
+      <code>history-cleanup-removed-task-metrics</code>,
+      <code>root-process-instance-start</code>,
+      <code>task-worker</code>
```
Comment (mboskamp) [inline content/reference/rest/metrics/get-metrics-interval.md:50]:
```suggestion
      <code>unique-task-workers</code>
```

File: content/reference/rest/metrics/get-metrics-sum.md [hunk 2]
```diff
@@ -40,11 +40,14 @@ GET `/metrics/{metrics-name}/sum`
...
       <code>job-locked-exclusive</code>,
       <code>executed-decision-elements</code>,
+      <code>executed-decision-instances</code>,
       <code>history-cleanup-removed-process-instances</code>,
       <code>history-cleanup-removed-case-instances</code>,
       <code>history-cleanup-removed-decision-instances</code>,
       <code>history-cleanup-removed-batch-operations</code>,
       <code>history-cleanup-removed-task-metrics</code>,
+      <code>root-process-instance-start</code>,
+      <code>task-worker</code>,
```
Comment (mboskamp) [inline content/reference/rest/metrics/get-metrics-sum.md:50]:
```suggestion
      <code>unique-task-workers</code>,
```

**Review by mboskamp — CHANGES_REQUESTED** [review]
Hi @superbeagle, thanks for adding the missing metrics. I think the task metric is called `unique-task-workers` instead of `task-worker` (see https://docs.camunda.org/manual/latest/user-guide/process-engine/metrics/ and https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/management/Metrics.java#L81.)

Otherwise it looks good to me :)

File: content/reference/rest/metrics/get-metrics-interval.md [hunk 3]
```diff
@@ -40,11 +40,14 @@ GET `/metrics`
...
       <code>executed-decision-elements</code>,
+      <code>executed-decision-instances</code>,
       <code>history-cleanup-removed-process-instances</code>,
       <code>history-cleanup-removed-case-instances</code>,
       <code>history-cleanup-removed-decision-instances</code>,
       <code>history-cleanup-removed-batch-operations</code>,
-      <code>history-cleanup-removed-task-metrics</code>
+      <code>history-cleanup-removed-task-metrics</code>,
+      <code>root-process-instance-start</code>,
+      <code>unique-task-workers</code>
```
Comment (tmetzke) [inline content/reference/rest/metrics/get-metrics-interval.md:50]: For future reference: the task worker metric is NOT supported in the interval endpoint, only in the sum endpoint

**Pull request merged**